core/application/impl/app_configuration_impl.cpp

```diff
@@ -775,6 +775,9 @@ namespace kagome::application {
           auto chain_spec = ChainSpecImpl::loadFrom(chain_spec_path_.native());
           auto path = keystorePath(chain_spec.value()->id());
 
+          auto ma_res = chain_spec.value()->bootNodes()[0];
+          listen_addresses_.emplace_back(ma_res);
```

Comment:
Is this a code used for testing?  
Remove this line, or search `bootNodes` by peer id

Reply:
This is a dev-mode code. I took first bootstrap node address as the address for dev-mode node.